### PR TITLE
Initial commit

### PR DESCRIPTION
**Pull request by danikkrapivnitskiy**

### CODE REVIEW

File: homework/english/HW Lecture 1 [hunk 1]
```diff
@@ -1,52 +1,54 @@
...
+2 Anyone who smokes 20 cigarettes a day can expect to have health problems.
+3 She doesn't like cooking, but today she is baking a pie.
+4 I  am finishing the project right now.
+5 I don’t listen to rock music very often.
+6 — What does your sister do ?
 — She is a programmer.
-7 My train (leaves at 5/is leaving at 5).
-8 He (never tidies up/is never tidying up) his room!!! (irritation)
+7 My train leaves at 5.
+8 He never tiding up his room!!! (irritation)
```
Comment (denisnaumovets) [inline homework/english/HW Lecture 1:15]: Correct is "tid**y**ing"

- [ ] add change

File: homework/english/HW Lecture 1 [hunk 2]
```diff
@@ -1,52 +1,54 @@
...
-1.2 Correct the grammar mistakes 
+1.2 Correct the grammar mistakes
 
-1 Are you usually working on Saturdays?
-2 Listen! Somebodys sings.
-3 Look! The sun shines.
-4 He (is not using) his car every day.
-5 — Where are you coming from?
+1 Do you usually work on Saturdays?
+2 Listen! Somebody sings.
```
Comment (denisnaumovets) [inline homework/english/HW Lecture 1:21]: Maybe correct is "Somebody is singing", because it is emotionality

File: homework/english/HW Lecture 1 [hunk 3]
```diff
@@ -1,52 +1,54 @@
...
+3 Look! The sun in shine.
```
Comment (denisnaumovets) [inline homework/english/HW Lecture 1:22]: Maybe correct is "The sun is shining", because it is emotionality

File: homework/english/HW Lecture 1 [hunk 4]
```diff
@@ -1,52 +1,54 @@
...
+4 He doesn't use his car every day.
+5 — Where did you come from?
```
Comment (denisnaumovets) [inline homework/english/HW Lecture 1:24]: Correct is "Where do you come from?"

File: homework/english/HW Lecture 1 [hunk 5]
```diff
@@ -1,52 +1,54 @@
...
 — I am from Canada.
-6 What happened? Why do you cry?
-7. She always says such silly things!!! 
-8. I don't go to work on Monday. I'm sick. 
+6 What happened? Why are you crying?
+7. She always saying such silly things!!!
```
Comment (denisnaumovets) [inline homework/english/HW Lecture 1:27]: Maybe correct is "is always saying", because it is emotionality

File: homework/english/HW Lecture 1 [hunk 6]
```diff
@@ -1,52 +1,54 @@
...
+8. I didn't go to work on Monday. I'm sick.
```
Comment (denisnaumovets) [inline homework/english/HW Lecture 1:28]: Incorrect time.

File: homework/english/HW Lecture 1 [hunk 7]
```diff
@@ -1,52 +1,54 @@
...
 1.3 Complete the sentences with am/is/are or do/does in positive or negative forms.
 
-1 He is a good musician, but he (…) perform very often.
-2 I (…) go to the gym on Mondays because I usually finish my work very late.
-3 Please turn off the laptop, you (…) using it.
-4 I (…) like tea, but today, I (…) drinking it.
-5 It's dark in your room. (…) you sleeping?
-6. She came to London to study English. But she (...) leaving in a student residence, she's staying at her aunt's. 
+1 He is a good musician, but he doesn't perform very often.
+2 I didn't go to the gym on Mondays because I usually finish my work very late.
```
Comment (denisnaumovets) [inline homework/english/HW Lecture 1:33]: Incorrect time.

**Review by denisnaumovets — COMMENTED** [review]
See notes

**Review by RyszardP — APPROVED** [review]

**Review by dk-t — APPROVED** [review]

**Pull request closed without merging**